modules/internal/ChapelArray.chpl

```diff
@@ -2027,7 +2030,28 @@ module ChapelArray {
     proc tail(): this._value.eltType {
       return this[this.domain.high];
     }
-
+    pragma "no doc"
+    proc resizeAllocRange(r: range, d2: domain, factor=arrayAsVecGrowthFactor, direction=1, grow=1) {
```

Comment:
If we are making changes, I would add "compilerAssert(d2.rank == 1)" for documentation,
with a comment that "we are operating only on 1-D arrays"